odd_collector/adapters/dbt/mappers/metadata.py

```diff
@@ -0,0 +1,15 @@
+from odd_models.models import MetadataExtension
+
+
+def _append_metadata_extension(metadata_list: list[MetadataExtension],
```

Comment:
What do you think
We can simplify that function as
```python
def _append_metadata_extension(metadata_list: List[MetadataExtension],
                               schema_url: str, metadata_dict: dict, excluded_keys: set = None
                               ) -> Optional[MetadataExtension]:
    if not metadata_dict:
        return None

    metadata_wo_none = {
        key: value
        for key, value in metadata_dict.items()
        if key not in excluded_keys
           and value is not None
           and value != ''
    }

    metadata_list.append(MetadataExtension(schema_url=schema_url, metadata=metadata_wo_none))
```

Pros:
* Easier to read
* return first if dict is None or empty
* We do not need to create intermediate objects as copy of metadata

Reply:
done